apps/system/js/bluetooth_transfer.js

```diff
@@ -644,15 +644,17 @@ var BluetoothTransfer = {
     var storage = navigator.getDeviceStorage(storageType);
     var getreq = storage.get(filePath);
```

Comment:
I retract previous statement, you are correct. I totally missed the check for the .then method elsewhere in the file.